tests/testthat/helper-server.R

```diff
@@ -86,3 +86,12 @@ hintr_server <- function(n_tries = 10, poll = 0.5, results_dir = tempdir()) {
 response_from_json <- function(x) {
   jsonlite::parse_json(httr::content(x, "text", encoding = "UTF-8"))
 }
+
+hintr_server_blocking <- function() {
```

Comment:
I was using this for debugging a server error - it just bring up hintr so that I could use browser within the server session